projects/storefrontstyles/scss/layout/page-templates/_landing-page-2.scss

```diff
@@ -5,6 +5,16 @@
   margin: auto;
 
   cx-page-slot {
+    // cx-banner {
```

Comment:
I've just pushed a change for this, which moves the focus related css rules to the banner component. that's where it belongs in my pov.